source/ranges.tex

```diff
@@ -4889,7 +4899,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Initializes \tcode{base_} with \tcode{views::all(std::forward<R>(r))} and
+Initializes \tcode{base_} with \tcode{views::all(std::forward<R>(r))}, and
```

Comment:
This added comma (and the one above) go against the convention.

Reply:
I added these for consistency with the constructor further above. I have no particular preference here other than consistency; please feel free to provide a PR to remove them.